src/Sylius/Bundle/PaymentBundle/Resources/config/services/provider.xml

```diff
@@ -43,5 +43,9 @@
             <argument type="tagged_locator" tag="sylius.http_response.provider.offline" index-by="action" />
             <tag name="sylius.payment_request.http_response.provider" gateway-factory="offline" />
         </service>
+
+        <service id="sylius.provider.webhook_payment" class="Sylius\Bundle\PaymentBundle\Provider\CompositeWebhookPaymentProvider">
```

Comment:
All services in this file marked as public -- does this service needs to be public?